erpnext/projects/doctype/time_log/time_log.js

```diff
@@ -25,16 +25,34 @@ frappe.ui.form.on("Time Log", "refresh", function(frm) {
 
 
 // set to time if hours is updated
-frappe.ui.form.on("Time Log", "hours", function(frm) {
-	if(!frm.doc.from_time) {
-		frm.set_value("from_time", frappe.datetime.now_datetime());
```

Comment:
this code was from Robert, I think he didn't consider  "Simplified Time Log" unchecked situation at that time. I have to check further.